codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/transformers/OperationNormalizer.kt

```diff
@@ -58,67 +49,42 @@ class OperationNormalizer(private val model: Model) {
         }
     }
 
-    private fun syntheticOutputShapes(
-        operation: OperationShape,
-        outputBodyFactory: StructureModifier
-    ): List<StructureShape> {
+    private fun syntheticOutputShapes(operation: OperationShape): List<StructureShape> {
```

Comment:
can this just be `StructureShape?` now?

Reply:
I think in this particular case, the code is actually cleaner with them being lists since the code that calls these functions is pretty clean that way, even if they only ever have one element. I would have to convert them at the call sites otherwise.